lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,20 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
+ * @function
+ * @public
+ *
+ * @description
+ * This function returns undefined if there was no last flash.
```

Comment:
Obviously the "retry" button is the only path after an error (which I guess makes the "the UUID will be cleared if a flash error occurs" statement valid), but its not part of the flash UUID core logic.